### PR TITLE
test: rebase tests 

### PR DESCRIPTION
**Pull request by witgaw**
Fix failing tests from `liquidity_provision` & `auctions` subfolders.

### CODE REVIEW

File: core/integration/features/auctions/5460-market-order-cannot-trade-due-to-auction.feature [hunk 1]
```diff
@@ -35,42 +35,6 @@ Feature: Test for issue 5460
       | party_r  | ETH   | 10000000000000000 |
       | party_r1 | ETH   | 10000000000000000 |
 
-  Scenario: 001 post market order
```
Comment (witgaw) [inline core/integration/features/auctions/5460-market-order-cannot-trade-due-to-auction.feature:38]: Not doing anything interesting.

File: core/integration/features/liquidity-provision/verify-CcySiska.feature [hunk 2]
```diff
@@ -178,39 +182,15 @@ Feature: check the impact from change of market parameter: market.liquidity.stak
...
-      | buy  | 900   | 1      |
-
-    #check position (party0 has no position)
-    Then the parties should have the following profit and loss:
-      | party  | volume | unrealised pnl | realised pnl |
-      | party1 | 50     | 0              | 0            |
-      | party2 | -50    | 0              | 0            |
+      | buy  | 900   | 10001  |
 
-    Then the parties should have the following margin levels:
```
Comment (witgaw) [inline core/integration/features/liquidity-provision/verify-CcySiska.feature:196]: the remainder of the test has nothing to do with checking impact of `market.liquidity.stakeToCcyVolume` parameter

File: core/integration/features/liquidity-provision/verify-CcySiska.feature [hunk 3]
```diff
@@ -241,39 +225,12 @@ Feature: check the impact from change of market parameter: market.liquidity.stak
...
       | sell | 1014  | 0      |
       | sell | 1010  | 1      |
       | sell | 1000  | 0      |
       | buy  | 1000  | 0      |
       | buy  | 990   | 1      |
       | buy  | 986   | 0      |
-      | buy  | 900   | 1      |
-
-    #check position (party0 has no position)
-    Then the parties should have the following profit and loss:
```
Comment (witgaw) [inline core/integration/features/liquidity-provision/verify-CcySiska.feature:254]: the remainder of the test has nothing to do with checking impact of `market.liquidity.stakeToCcyVolume` parameter

**Review by witgaw — COMMENTED** [review]
Ready for review.

Deleted `core/integration/features/auctions/2952-opening-auction-extension-csv.feature` as it's got nothing to do with the bug its supposedly refering and scenario desciption doesn't say anything suggesting it's useful for anything.

**Review by Jiajia-Cui — APPROVED** [review]
LGTM, will raise a bug in a separate issue

**Pull request merged**